networkx/algorithms/bipartite/tests/test_redundancy.py

```diff
@@ -25,6 +25,12 @@ def test_redundant_nodes():
         assert redundancy[v] == 1
 
 
+def test_redundant_nodes_2():
```

Comment:
Just a nit pick about the name.

Could you rename this to something like `test_redundant_nodes_subset`